kornia/constants.py

```diff
@@ -53,6 +54,42 @@ class SamplePadding(ConstantBase, Enum, metaclass=EnumMetaFlags):
     REFLECTION = 2
 
 
+class DType(ConstantBase, Enum, metaclass=EnumMetaFlags):
+    INT64 = 0
+    FLOAT16 = 1
+    FLOAT32 = 2
+    FLOAT64 = 3
+
+    @classmethod
+    def get(cls, value: Union[str, int, torch.dtype, Tensor, T]) -> T:  # type: ignore
```

Comment:
From torch

Reply:
@edgarriba it accepts more variants than from torch.